@xen-orchestra/lite/src/views/settings/SettingsView.vue

```diff
@@ -49,20 +49,39 @@
         </UiKeyValueRow>
       </UiKeyValueList>
     </UiCard>
-    <UiCard class="group">
-      <UiCardTitle>{{ $t("display") }}</UiCardTitle>
-      <UiKeyValueList>
-        <UiKeyValueRow>
-          <template #key>{{ $t("appearance") }}</template>
-          <template #value>
-            <FormSelect v-model="colorMode">
-              <option value="auto">{{ $t("theme-auto") }}</option>
-              <option value="dark">{{ $t("theme-dark") }}</option>
-              <option value="light">{{ $t("theme-light") }}</option>
-            </FormSelect>
-          </template>
-        </UiKeyValueRow>
-      </UiKeyValueList>
+    <UiCard class="group appearance">
+      <UiCardTitle>{{ $t("appearance") }}</UiCardTitle>
+      <div class="options">
+        <div
+          v-for="colorModeOption in colorModeOptions"
+          :key="colorModeOption"
+          class="option"
+          :class="{ selected: colorMode === colorModeOption }"
+          @click="() => (colorMode = colorModeOption)"
```

Comment:
Although it is valid, it is not necessary to wrap the action in an additional function here:

```suggestion
          @click="colorMode = colorModeOption"
```